### PR TITLE
CLOUDP-97432: live migration validation job status command

### PR DESCRIPTION
**Pull request by colm-quinn**
## Proposed changes

Support for "Return Status of One Specific Validation Job" [Docs](https://docs.atlas.mongodb.com/reference/api/live-migration/return-one-specific-validation-job/)

_Jira ticket:_ CLOUDP-97428

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [ ] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

Command Help:
```
bin/mongocli atlas lm validation describe --help --profile=atlas-dev                  
Your API Key must have the Organization Owner role to successfully run this command.

Usage:
  mongocli atlas liveMigrations validation describe [flags]

Aliases:
  describe, get

Flags:
  -h, --help                  help for describe
  -o, --output string         Output format.
                              Valid values: json|json-path|go-template|go-template-file
      --projectId string      Project ID to use. Overrides configuration file or environment variable settings.
      --validationId string   Unique 24-hexadecimal digit string that identifies the validation job.
```

Command output:
```
bin/mongocli atlas lm validation describe --validationId=6141c02eadd36135e7414ac3 -o=json --profile=atlas-dev
{
  "_id": "6141c02eadd36135e7414ac3",
  "groupId": "6137493677e94f0e973d1a0f",
  "status": "PENDING",
  "sourceGroupId": "611e68cb086adf5cda2e6253"
}
```

### CODE REVIEW

File: internal/cli/atlas/livemigrations/options/live_migrations_opts.go [hunk 1]
```diff
@@ -138,6 +139,7 @@ func (opts *LiveMigrationsOpts) GenerateFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&opts.DestinationClusterName, flag.ClusterName, "", usage.LiveMigrationDestinationClusterName)
 	cmd.Flags().StringSliceVar(&opts.MigrationHosts, flag.LiveMigrationHost, []string{}, usage.LiveMigrationHostEntries)
 	cmd.Flags().BoolVar(&opts.DestinationDropEnabled, flag.LiveMigrationDropCollections, false, usage.LiveMigrationDropCollections)
+	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
```
Comment (colm-quinn) [inline internal/cli/atlas/livemigrations/options/live_migrations_opts.go:142]: [flyby] Allowing json output for the live migration commands

File: internal/cli/atlas/livemigrations/options/live_migrations_opts.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ import (
 )
 
 type LiveMigrationsOpts struct {
+	cli.OutputOpts
```
Comment (fmenezes) [inline internal/cli/atlas/livemigrations/options/live_migrations_opts.go:31]: @colm-quinn If you are adding it here it is better to remove it from the create commands as well

**Comment by fmenezes** [comment]
Overall looks good, just blocking on one comment

**Review by fmenezes — CHANGES_REQUESTED** [review]
Overall looks good, just blocking on one comment

File: internal/cli/atlas/livemigrations/options/live_migrations_opts.go (hunk 2 above)
Comment (colm-quinn) [reply internal/cli/atlas/livemigrations/options/live_migrations_opts.go:31]: Nice catch! Will do!

**Review by melissamahoney-mongodb — APPROVED** [review]

**Review by fmenezes — APPROVED** [review]
code looks good

**Pull request merged**